manifests/deploy/service.pp

```diff
@@ -0,0 +1,31 @@
+# == Class: openvpn::deploy::service
+#
+# Base profile
+#
+# === Parameters
+#
+# None
+#
+# === Variables
+#
+# None
+#
+# === Examples
+#
+#  include openvpn::deploy::service
+#
+# === Authors
+#
+# Phil Bayfield https://bitbucket.org/Philio/
+#
+
+class openvpn::deploy::service {
+
+  service { 'openvpn':
```

Comment:
this is also redundant with https://github.com/voxpupuli/puppet-openvpn/blob/master/manifests/service.pp ?

Reply:
i am not shure, if i should/could reuse the server part for the client here too? (same as for install)